### PR TITLE
Update Galaxy-Workflow-Persist-Seq-cell_cycle.ga to add filter tools

### PR DESCRIPTION
**Pull request by anilthanki**
Scanpy scrublet was failing with low number of genes per cell, to avoid this kind of failure, added `filter_gene` and `filter_cell` before `scanpy_scrublet`

### CODE REVIEW

File: SC-workflow/Persist-Seq/Galaxy-Workflow-Persist-Seq-cell_cycle.ga [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "a_galaxy_workflow": "true",
-    "annotation": "Persist-Seq v0.3.6, Scanpy 1.9.3 (cell cycle)",
+    "annotation": "Persist-Seq v0.3.7, Scanpy 1.9.3 (cell cycle)",
```
Comment (pcm32) [inline SC-workflow/Persist-Seq/Galaxy-Workflow-Persist-Seq-cell_cycle.ga]:
```suggestion
    "annotation": "Persist-Seq v0.4.0, Scanpy 1.9.3 (cell cycle)",
```
Sufficiently large change I would say.

File: SC-workflow/Persist-Seq/Galaxy-Workflow-Persist-Seq-cell_cycle.ga [hunk 2]
```diff
@@ -14,7 +14,7 @@
         }
     ],
     "format-version": "0.1",
-    "name": "Persist-Seq v0.3.6",
+    "name": "Persist-Seq v0.3.7",
```
Comment (pcm32) [inline SC-workflow/Persist-Seq/Galaxy-Workflow-Persist-Seq-cell_cycle.ga]:
```suggestion
    "name": "Persist-Seq v0.4.0",
```

**Review by pcm32 — COMMENTED** [review]
Great! I sent some other comments via slack, these JSON files are impossible to read humanely, hehehe. Something for the galaxy community to improve.

**Comment by anilthanki** [comment]
> Great! I sent some other comments via slack, these JSON files are impossible to read humanely, hehehe. Something for the galaxy community to improve.

fixed here [e1b49d6](https://github.com/AstraZeneca/persist-seq-galaxy-tools-setup/pull/13/commits/e1b49d66c38f280682f941e7c50fa3b4e2e05ec7)

**Comment by pcm32** [comment]
LGTM, great work!

**Pull request merged**